File: Exercise/Easy/10. Maximum of Two Numbers/Code/Maximum of Two Numbers.cpp
#include <iostream>

using namespace std;

void maxNumbers(int num1, int num2) {
    if (num1 > num2) {
        cout << num1 << endl;
    }
    else {
        cout << num2 << endl;
    }
}

int main() {
    int num1, num2;
    cin >> num1 >> num2;

    maxNumbers(num1, num2);

    return 0;
}

File: Exercise/Easy/10. Maximum of Two Numbers/Code/Maximum of Two Numbers.java
import java.util.Scanner;

public class MaxNumbers {
    public static void maxNumbers(int num1, int num2) {
        if (num1 > num2) {
            System.out.println(num1);
        }
        else {
            System.out.println(num2);
        }
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int num1 = scanner.nextInt();
        int num2 = scanner.nextInt();
        scanner.close();

        sumNumbers(num1, num2);
    }
}


